i18n/en.json

```diff
@@ -113,13 +113,21 @@
     "jobs_failed": "{jobCount, plural, other {# failed}}",
     "library_created": "Created library: {library}",
     "library_deleted": "Library deleted",
+    "library_details": "Library details",
+    "library_exclusion_pattern_added": "Exclusion pattern added",
+    "library_exclusion_pattern_deleted": "Exclusion pattern deleted",
+    "library_exclusion_pattern_edited": "Exclusion pattern edited",
+    "library_import_path_added": "Import path added",
+    "library_import_path_deleted": "Import path deleted",
```

Comment:
I don't think there is a lot of value in being so granular in these success messages. I think it would be better to have one general one for libraries like `Saved library`. I am also fine _just_ using `toastManager.success()` or `toastManager.success($t('saved'))`.